en/documentation/installation/index.md

```diff
@@ -334,6 +335,18 @@ manage Rubies installed by [ruby-install](#ruby-install) or even built
 from source.
 
 
+### mise-en-place
+{: #mise-en-place}
+
+[mise-en-place][mise-en-place] allows you to switch between multiple Rubies.
+It uses [ruby-build](#ruby-build) under the hood but can also be configured to use [ruby-install](#ruby-install)
+—neither of which needs to be manually installed first. mise-en-place is a polyglot
```

Comment:
not sure if this is too wordy, I felt like if I mention ruby-build/ruby-install that may confuse users thinking they need to first install those which they don't (in fact mise can't even be configured to use an existing ruby-install/ruby-build)